### PR TITLE
Event edit about

### PR DESCRIPTION
**Pull request by hokansom**
This is the event edit page. It includes two other branches that are still awaiting code review as part of it. We will add in the edit functionality for title and date later on (#98 ). I started adding some functionality, but the rest will be completed as part of issue #82.  

@jacson10e @jam1023 I did add some specific code that you will need for the event view page. Specifically, this PR has code for getting the event id from the url which you will need to incorporate on your branch. Also, I made some changes to page layouts, so you will want to update your branches as well (for the event view page). 

As a general note, if you see anything in the code that you don't understand, please leave a comment. 

Screen Shot 2020-11-02 at 8 49 53 PM

### CODE REVIEW

**Review by jam1023 — COMMENTED** [review]
Looks great!

Is this the code you were talking about that gets the event id from URL? 

  useEffect(() => {
    const params = new URLSearchParams(location.search);
    let eventId = params.get("id");
    if (eventId) {
      loadPageData(eventId);
    }
    // eslint-disable-next-line
  }, [location.search]);

I'm just commenting instead of approving so @jacson10e can do his review too.

Thanks!

**Comment by hokansom** [comment]
@jam1023 Yes, that's the code I was talking about! You'll want to make a minor change (the bold part). If there isn't an eventId then there's nothing to load, so we should probably send them back to the home page. 

> 
> useEffect(() => {
> const params = new URLSearchParams(location.search);
> let eventId = params.get("id");
> if (eventId) {
> loadPageData(eventId);
> } **else {**
>  _**//redirect to home page since there isn't an eventId**_
> **}**
> // eslint-disable-next-line
> }, [location.search]);

File: client/src/components/EventEdit/Header/index.js [hunk 1]
```diff
@@ -53,17 +50,17 @@ const Header = ({
...
+    <Moment format="MM/DD/YYYY">{date}</Moment>
+  );
 
   return (
     <Container {...props}>
       <Stack>
         <TitleContainer>{title}</TitleContainer>
-        <Spacer />
         <DateContainer>{effectiveDate}</DateContainer>
-        <Spacer />
```
Comment (josiahsrc) [inline client/src/components/EventEdit/Header/index.js:66]: Oh whoops, I forgot to delete these. Thanks for catching that

**Review by josiahsrc — APPROVED** [review]
Looks great! The UI is spot on, nice job

**Pull request merged**